### PR TITLE
fix/victor connection banner

### PR DESCRIPTION
**Pull request by VicJohn447**
using a snackbar to display connection message instead of an alert

### CODE REVIEW

File: layouts/default.vue [hunk 1]
```diff
@@ -4,22 +4,18 @@
...
-        <v-alert :value="$nuxt.isOffline" type="error" dismissible class="ma-0">
-          <!-- eslint-disable-next-line -->
-          You are currently offline. Please check your internet connection.
-        </v-alert>
         <v-container :fill-height="isIndex" fluid class="pa-0 mx-0">
           <nuxt />
         </v-container>
         <v-snackbar
-          v-model="snackbar"
+          v-model="$nuxt.isOffline"
```
Comment (JeremiahPinto) [inline layouts/default.vue]: This removes the previous trigger which lets the user know if they can't use geolocation. You will have to add the `$nuxt.isOffline` to the computed property as was previously done.

**Review by JeremiahPinto — CHANGES_REQUESTED** [review]
Please fix the issue I have outlined next to the code. I know that the code that uses it is commented out for now, but it will be used when we have the backend running

File: layouts/default.vue [hunk 2]
```diff
@@ -19,7 +15,16 @@
           multi-line
         >
           {{ snackbarText }}
-          <v-btn @click="snackbar = false" flat>Close</v-btn>
+          <v-btn @click="snackbar = false" color="white" text flat>Close</v-btn>
+        </v-snackbar>
+        <v-snackbar
+          v-model="$nuxt.isOffline"
+          :timeout="offlineTimeout"
+          :color="snackbarColour"
```
Comment (JeremiahPinto) [inline layouts/default.vue]: Doesn't need a dynamic colour if this never changes

File: layouts/default.vue [hunk 3]
```diff
@@ -19,7 +15,16 @@
...
+          bottom
+          multi-line
+        >
+          {{ offlineText }}
```
Comment (JeremiahPinto) [inline layouts/default.vue]: Don't need a variable for static text

**Review by JeremiahPinto — COMMENTED** [review]
Why is the entire file marked as changed? Maybe check your line endings (Should be `LF`).

**Comment by Drewbi** [comment]
This PR Is disconnected from the branch for some reason. Use the other one to merge.

**Pull request closed without merging**